# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../musepack-tools/undef_irix_esd.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- sv7/mpp.h.orig	2004-11-20 17:03:58.000000000 +0200
+++ sv7/mpp.h	2006-09-25 22:02:29.000000000 +0300
@@ -40,7 +40,7 @@
 // If your Operating System supports the Enlightenment Sound Daemon you can output to /dev/esd and
 // instead of writing a file the program plays the file via this sound device.
 // you also must link the libesd library, so maybe you also must edit the Makefile
-#define USE_ESD_AUDIO
+#undef USE_ESD_AUDIO
 
 #endif
 
@@ -54,7 +54,7 @@
 // Sound support for SGI Irix
 // If you have a SGI Workstation running IRIX, you can output to /dev/audio and
 // instead of writing a file the program plays the file via this sound device.
-#define USE_IRIX_AUDIO
+#undef USE_IRIX_AUDIO
 
 // Audio support for Windows (WAVE OUT) (only Windows)
 // If you have a Windows based system and if you want also play files directly instead of only writing audio files,
